maps-compose/src/main/java/com/google/maps/android/compose/streetview/StreetView.kt

```diff
@@ -23,14 +37,17 @@ import androidx.lifecycle.Lifecycle
 import androidx.lifecycle.LifecycleEventObserver
 import com.google.android.gms.maps.StreetViewPanoramaOptions
 import com.google.android.gms.maps.StreetViewPanoramaView
-import com.google.android.gms.maps.model.StreetViewPanoramaCamera
 import com.google.android.gms.maps.model.StreetViewPanoramaOrientation
 import com.google.maps.android.compose.disposingComposition
 import com.google.maps.android.ktx.MapsExperimentalFeature
 import com.google.maps.android.ktx.awaitStreetViewPanorama
 
 /**
- * A composable for displaying a Street View for a given location.
+ * A composable for displaying a Street View for a given location. A location might not be available for a given
+ * set of coordinates. We recommend you to check our sample on [StreetViewActivity] using our utility function
+ * in [StreetViewUtils] to manage non-existing locations.
```

Comment:
using the `fetchStreetViewMetadata` utility function in `maps-compose-utils`

Reply:
What will be the destination link in the docs page for the "[StreetViewUtils]" mention? I don't see a StreetViewUtils page in the docs https://googlemaps.github.io/android-maps-compose/maps-compose-utils/index.html

Reply:
Issue causing StreetViewUtils page to not show up on `android-maps-utils` refdocs. https://github.com/googlemaps/android-maps-utils/issues/1248

Per offline conversation we are expecting [StreetViewUtils] to point to the corresponding page in `android-maps-utils` refdocs.